### PR TITLE
fix(core): make the projector not exit on asset name error

### PR DESCRIPTION
**Pull request by iccicci**
# Context

The `asset-projector` exit with error while projecting.

# Proposed Solution

Wrapped the line throwing in a `try / catch` block.

### CODE REVIEW

**Review by VanessaPC — APPROVED** [review]
I think this is a good solution in that we catch the asset name in the logs and let docker continue. 
Nice solution

File: packages/core/src/Asset/NftMetadata/fromMetadatum.ts [hunk 1]
```diff
@@ -121,7 +121,11 @@ export const fromMetadatum = (
   const assetId = Cardano.AssetId.fromParts(asset.policyId, asset.name);
 
   if (version === '1.0' && !name) {
-    name = AssetName.toUTF8(asset.name);
+    try {
+      name = AssetName.toUTF8(asset.name);
+    } catch (error) {
+      logger.warn(error);
```
Comment (mkazlauskas) [inline packages/core/src/Asset/NftMetadata/fromMetadatum.ts:127]: We shouldn't continue in this case, as metadata is invalid

```suggestion
      logger.warn(error);
      return null;
```

**Review by mkazlauskas — CHANGES_REQUESTED** [review]
What is the problematic asset id? I suspect it may be for cip67-encoded asset :thinking:

File: packages/core/src/Asset/NftMetadata/fromMetadatum.ts (hunk 1 above)
Comment (iccicci) [reply packages/core/src/Asset/NftMetadata/fromMetadatum.ts:127]: The `return null` is already in the next line

**Review by mkazlauskas — APPROVED** [review]

**Pull request merged**